### PR TITLE
add winc vSphere nightly support for 4.14-4.17

### PR DESCRIPTION
**Pull request by weinliu**

### CODE REVIEW

**Comment by weinliu** [comment]
https://github.com/openshift/release/pull/58677/files for 4.16 merged
cron jobs works as expected https://reportportal-openshift.apps.ocp-c1.prod.psi.redhat.com/ui/#prow/launches/1076/656973

**Comment by weinliu** [comment]
@rrasouli @liangxia could you help?
/uncc @Xia-Zhao-rh @rioliu-rh

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 1]
```diff
@@ -3048,6 +3048,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-upi-zones
+- as: vsphere-ipi-ovn-winc-f7
+  cron: 3 18 7,16,23,30 * *
```
[the remark this answers is not in the document]
Comment (weinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml]: @liangxia got you thanks.  4.14 fequency reduced by 50%,  aslo get 4.15 and 4.17 time adjusted

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 2]
```diff
@@ -3048,6 +3048,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-upi-zones
+- as: vsphere-ipi-ovn-winc-f7
+  cron: 3 18 16,30 * *
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:3052]: Frequency `fN` (every 7 days) mismatch with the cron entry ( every 14 days).
You can update the frequency `fN`, then use the [shell](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh) script to generate the cron entry, or the [python](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/tools/update-cron-entries.py) one to update them in batch.

Comment (weinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:3052]: Got you, updated to f14. Thanks to the tools advice

**Comment by weinliu** [comment]
@liangxia could you help take another look?

```
>  sh generate-cron-entry.sh vsphere-ipi-ovn-winc-f14 openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml
-n cron: 
34 9 15,29 * *

>  sh generate-cron-entry.sh vsphere-ipi-ovn-winc-f14 openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml
-n cron: 
52 8 16,30 * *
```

**Comment by liangxia** [comment]
Unfortunately, I can see 4.14 and 4.17 both runs on day 16 and day 30.

**Comment by liangxia** [comment]
> > Unfortunately, I can see 4.14 and 4.17 both runs on day 16 and day 30.
> 
> ```
> > sh generate-cron-entry.sh vsphere-ipi-ovn-winc-f7 openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml
> -n cron: 
> 36 1 2,9,16,25 * *
> ```
> 
> @liangxia how about this time?

Looks like 4.14 and 4.17 both runs on day 16 ? That's why generally we don't update cron entry manually.

**Pull request merged**